drivers/bluetooth/hci/Kconfig

```diff
@@ -109,6 +109,21 @@ config BT_STM32_IPM_RX_STACK_SIZE
 	depends on BT_STM32_IPM
 	default 512
 
+if BT_H4
+
+config BT_CYW43XXX
```

Comment:
You could convert this to a `menuconfig` and then include `source "drivers/bluetooth/hci/Kconfig.infineon` just under it, then the extra option would nest under that in `west build -t menuconfig`. Not sure how much the Bluetooth maintainers care but I think it'd be a bit cleaner like that.

Reply:
Yes with menuconfig it looks more structured
![image](https://user-images.githubusercontent.com/46566584/222563557-a58630f3-f0d0-4c45-8b86-197e7ce7b50e.png)